tests/test_committees.py

```diff
@@ -373,6 +373,20 @@ def test_committee_date_filters(self):
             )
         )
 
+    def test_filter_by_sponsor_candidate_ids(self):
+        sponsor_candidate_ids = ['H001']
```

Comment:
@hcaofec would it be a more meaningful filter test if we had more than one row? Maybe one for `H002`?